include/Core/Posts/PostsMetabox.php

```diff
@@ -251,4 +251,67 @@ protected function delete_posts_by_id( $post_ids, $force_delete ) {
 
 		return count( $post_ids );
 	}
+
+	/**
+	 * Generate display name from post type and status.
+	 *
+	 * @static
+	 *
+	 * @param string $str
+	 *
+	 * @return string Label
+	 */
+	public static function display_post_type_status( $str ) {
```

Comment:
Make this function as protected and non-static.

You can also remove this function from BDUtil class